docs/registering-samples.md

```diff
@@ -5,20 +5,22 @@ title: Registering Samples
 
 In the ARGO Data Platform, clinical and molecular data objects are assigned ARGO Identifiers (**ARGO IDs**) used to track the data through the Platform. Each **Donor**, **Specimen**, and **Sample** entity will be assigned an **ARGO ID**, mapping to your program's internal identifiers.
 
-It is important that the relationships between entities are maintained across submissions, as they are fundamental to data tracking and integrity across the ARGO Data Platform.
+It is important that the relationships between entities are maintained across all data submissions and formats, as they are fundamental to data integrity across the ARGO Data Platform.
```

Comment:
I'm not sure what you mean by "and formats"?

Reply:
whether submitting clinical through ui (eventually api) and genomic thru api clients

Reply:
can say 
across all data submissions submitting through any modes of submission

Reply:
sounds kinda gross.  ill remove `and formats` for now instead.